doc/source/timeseries.rst

```diff
@@ -426,10 +426,11 @@ Custom Frequency Ranges
 .. warning::
 
    This functionality was originally exclusive to ``cdate_range``, which is
-   deprecated as of version 0.21.0 in favor of ``bdate_range``.  Note that
-   ``cdate_range`` only utilizes the ``weekmask`` and ``holidays`` parameters
-   when custom business day, 'C', is passed as the frequency string. Support has 
-   been expanded with ``bdate_range`` to work with any custom frequency string.
+   deprecated as of version 0.21.0 (deleted in 0.24.0) in favor of
+   ``bdate_range``. Note that ``cdate_range`` only utilizes the ``weekmask``
+   and ``holidays`` parameters when custom business day, 'C', is passed
+   as the frequency string. Support has been expanded with ``bdate_range``
+   to work with any custom frequency string.
```

Comment:
Do we still need to reference `cdate_range` in the docs?  Or can we rewrite to remove it?

Reply:
Im thinking of just removing that warning altogether since it just mentions some subtleties of `cdate_range`. Alternatively potential rewrite could look like:
`
This functionality was originally exclusive to ``cdate_range``, which is
deprecated as of version 0.21.0 (deleted in 0.24.0) in favor of
``bdate_range`` which works with any custom frequency string.
`
Thoughts?

Reply:
I would just remove it altogether.

Reply:
Makes sense! updated